wcc-contentful-app/lib/wcc/contentful/model/menu_button.rb

```diff
@@ -1,16 +1,50 @@
 # frozen_string_literal: true
 
 class WCC::Contentful::Model::MenuButton < WCC::Contentful::Model
-  validate_field :text, :String, :required
+  validate_field :text, :String
   validate_field :icon, :Asset, :optional
+  validate_field :material_icon, :String
   validate_field :external_link, :String, :optional
-  validate_field :link, :Link, :optional, link_to: 'page'
+  validate_field :link, :Link, :optional, link_to: %w[page page-v2]
+  validate_field :section_link, :Link, :optional,
+    link_to: %w[
```

Comment:
Get rid of the `link_to` validation here because we don't want to pre-define what kind of sections an app will have.